src/components/Main/Navigation/ChannelList/ChannelElementHash.vue

```diff
@@ -102,29 +103,63 @@ export default defineComponent({
     }
   }
   &[data-container-type='parent'] {
+    &:hover {
+      &::before {
+        content: '';
+        border-radius: 4px;
+        display: block;
+        position: absolute;
+        top: -4px;
+        bottom: -4px;
+        left: -4px;
+        right: -4px;
+      }
+    }
     &[data-is-opened] {
       color: $theme-background-secondary;
       background: $theme-ui-primary;
+      &:hover::before {
+        background: $theme-ui-secondary;
+        opacity: 0.5;
+      }
       &[data-is-inactive] {
         background: $theme-ui-secondary;
       }
       &[aria-selected='true'] {
         @include background-accent-primary;
+        &:hover::before {
+          background: $theme-accent-primary;
+          opacity: 0.7;
+        }
       }
     }
     &:not([data-is-opened]) {
       @include color-ui-primary;
       border-color: $theme-ui-primary;
-      &[data-is-inactive] {
-        @include color-ui-secondary;
-        border-color: $theme-ui-secondary;
-      }
-      &[data-has-notification-on-child] {
-        border-color: $theme-accent-notification;
+      &:not([area-selected]) {
+        &:hover::before {
+          background: $theme-ui-primary;
+          opacity: 0.2;
+        }
+        &[data-is-inactive] {
+          @include color-ui-secondary;
+          border-color: $theme-ui-secondary;
+        }
+        &[data-has-notification-on-child] {
+          border-color: $theme-accent-notification;
+          &:hover::before {
+            background: $theme-accent-notification;
+            opacity: 0.1;
+          }
+        }
       }
```

Comment:
```suggestion
      &[data-is-inactive] {
        @include color-ui-secondary;
        border-color: $theme-ui-secondary;
        &:hover::before {
          background: $theme-ui-primary;
          opacity: 0.2;
        }
      }
      &[data-has-notification-on-child] {
        border-color: $theme-accent-notification;
        &:hover::before {
          background: $theme-accent-notification;
          opacity: 0.1;
        }
      }
```
このほうがわかりやすそうです
それと`[aria-selected]`は`false`になることがあっても存在しないことはたぶんないのでうまく動かないです